os/android/iahwc2.cpp

```diff
@@ -143,14 +143,14 @@ HWC2::Error IAHWC2::Init() {
   property_get("board.hwc.scaling.mode", value, "0");
   scaling_mode_ = atoi(value);
   switch (scaling_mode_) {
+    case 0:
+      ALOGI("HWC Scaling Mode High Quality");
+      break;
     case 1:
       ALOGI("HWC Scaling Mode Fast");
       break;
-    case 2:
-      ALOGI("HWC Scaling Mode High Quality");
```

Comment:
please keep original order, this will match the order with libva, else it may introduce confuse.

Reply:
If this, it will become:
==============
case 0:
case 2:
    High quality mode
    break;
case 1:
    Fast mode
    break;
default:
    unsupport mode
    break; 
=============
That means 0&2 all will get into high quality mode. That's also a little confused:)
BTW, if don't set board.hwc.scaling.mode in android environment, property_get will return 0.

Reply:
case 2:
High quality mode
break;
case 1:
Fast mode
break;
default:
unsupport mode
break;